### ISSUE TITLE
[Accessibility]: Global nav bar: "Code search" dropdown doesn't have obvious focus styles

### ISSUE THREAD

**Issue by umpox** [sourcegraph/sourcegraph#35113]
### Audit type

Keyboard navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34411

### Problem description

Keyboard navigate to open the "Code search" dropdown. No focus outline on menu (even though keyboard focus is there):

<img width="265" alt="image" src="https://user-images.githubusercontent.com/9516420/167413524-a7556711-df4f-40ea-812d-bc5c48747c81.png">

### Expected behavior

Expected to see an obvious style that indicates where the keyboard focus is.

For example, we can easily see when the "Code search" link is highlighted.

<img width="165" alt="image" src="https://user-images.githubusercontent.com/9516420/167413838-0939f852-e12d-4268-9218-125e2f2e6817.png">

We should have the same outline around the entire dropdown

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by umpox** [comment]
Might be fixed by https://github.com/sourcegraph/sourcegraph/issues/36350

**Comment by umpox** [comment]
Closed by https://github.com/sourcegraph/sourcegraph/issues/36350!
